samples/esx_create_ds.py

```diff
@@ -0,0 +1,114 @@
+import requests
+
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+
+def get_args():
+    """
+    Supports the command-line arguments listed below.
+    """
+    parser = argparse.ArgumentParser(
+        description='Arguments for talking to vCenter')
+
+    parser.add_argument('-s', '--host',
+                        required=True,
+                        action='store',
+                        help='vSpehre service to connect to')
+
+    parser.add_argument('-o', '--port',
+                        type=int,
+                        default=443,
+                        action='store',
+                        help='Port to connect on')
+
+    parser.add_argument('-u', '--user',
+                        required=True,
+                        action='store',
+                        help='User name to use')
+
+    parser.add_argument('-p', '--password',
+                        required=False,
+                        action='store',
+                        help='Password to use')
+
+    parser.add_argument('--uuid',
+                        required=True,
+                        action='store',
+                        help='uuid of disk')
+
+    parser.add_argument('--dsname',
+                        required=False,
+                        action='store',
+                        help='name of datastore')
+
+    args = parser.parse_args()
+
+    if not args.password:
+        args.password = getpass.getpass(
+            prompt='Enter password')
+
+    return args
+
+
+def createDatastore(si, uuid, dsname):
+    dp = "/vmfs/devices/disks/" + str(uuid)
+    content = si.RetrieveContent()
+    container = content.viewManager.CreateContainerView(
+                                               content.rootFolder,
+                                               [vim.HostSystem],
+                                               True)
+    for esx_host in container.view:
+        try:
+            hostdssystem = esx_host.configManager.datastoreSystem
+            dssystem = vim.host.DatastoreSystem
+            vmfs_ds_options = dssystem.QueryVmfsDatastoreCreateOptions(
+                                                       hostdssystem, dp, 5)
+            if dsname is not None:
+                vmfs_ds_options[0].spec.vmfs.volumeName = dsname
+
+            new_ds = dssystem.CreateVmfsDatastore(hostdssystem,
+                                                  vmfs_ds_options[0].spec)
+        except vim.fault.NotFound:
+            print "Not found"
+        except vim.fault.HostConfigFault:
+            print "host config fault"
+        except vmodl.fault.NotSupported:
+            print "Not supported"
+        except Exception as e:
+            print "Unexpected error:", e
+        else:
+            print "Datastore created"
+
+
+def main():
+    args = get_args()
+
+    # connect this thing
+    si = connect.SmartConnect(
```

Comment:
My question is on Removing Datastores using pyvmomi: 

I am writing a script that mounts a datastore and then unmounts it again. 

While mounting I am giving the datastore a name eg. "test". 

But on vCenter, its getting created with the name as UUID. So, when I say RemoveDatastore fand pass "test", it says no such datastore found. 

Please help!